src/LinuxTracing/LinuxTracingUtilsTest.cpp

```diff
@@ -405,4 +405,13 @@ TEST(FindFunctionsThatUprobesCannotInstrumentWithMessages, ModuleNotInMaps) {
                                "not loaded by the process."));
 }
 
+TEST(RetrieveInitialTidToRootNamespaceTidMapping, TrivialMapFromTestProcess) {
```

Comment:
That's a neat test! I am wondering if we also want to have a test where we read the "string". I'll keep it up to you.

Reply:
I think it is good enough as is. I don't want to split these 30 lines into three functions to test the internals.